src/components/ZoumCalculator.js

```diff
@@ -0,0 +1,111 @@
+import './Calculator.css';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+export default function Affiche() {
+  const [calculatorValue, setCalculatorValue] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (value) => {
+    const newValue = calculate(calculatorValue, value);
+    setCalculatorValue(newValue);
+  };
+
+  const displayValue = calculatorValue.next || calculatorValue.total || '0';
+
+  return (
+    <div>
+      <div className="affichage">{displayValue}</div>
+      <div>
+        <button type="button" onClick={() => handleClick('AC')}>
+          AC
+        </button>
+        <button type="button" onClick={() => handleClick('+/-')}>
+          +/-
+        </button>
+        <button type="button" onClick={() => handleClick('%')}>
+          %
+        </button>
+        <button
+          className="ColorBtn"
+          type="button"
+          onClick={() => handleClick('÷')}
+        >
+          ÷
+        </button>
+      </div>
+      <div>
+        <button type="button" onClick={() => handleClick('7')}>
+          7
+        </button>
+        <button type="button" onClick={() => handleClick('8')}>
+          8
+        </button>
+        <button type="button" onClick={() => handleClick('9')}>
+          9
+        </button>
```

Comment:
- [ ] Just like the previous reviewer said, I think it will be more ideal that your code is dry by trying to put all this repetition in an array, then looping through them, something like

```
const buttonKeys = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '=']

<div>
 {buttonKeys.map((buttonkey) => {
                return <button type='button' key={buttonkey.id} onClick={() => handleClick(buttonkey)}>
                      {buttonkey}
                </button>
            })}
</div>
```

See how that long line of code, becomes that short, whoops 😄.
Then the styling can be done using css selectors, you can do that with logic also, but with CSS selectors is less painful 🤣.